cmd/basecli/commands/cmds.go

```diff
@@ -10,48 +10,56 @@ import (
 
 	"github.com/tendermint/light-client/commands"
 	txcmd "github.com/tendermint/light-client/commands/txs"
+	ctypes "github.com/tendermint/tendermint/rpc/core/types"
+	cmn "github.com/tendermint/tmlibs/common"
 
 	btypes "github.com/tendermint/basecoin/types"
 )
 
-/*** Here is the sendtx command ***/
+/******** SendTx *********/
 
+// SendTxCmd is CLI command to send tokens between basecoin accounts
 var SendTxCmd = &cobra.Command{
 	Use:   "send",
 	Short: "send tokens from one account to another",
 	RunE:  doSendTx,
 }
 
+//nolint
 const (
-	ToFlag       = "to"
-	AmountFlag   = "amount"
-	FeeFlag      = "fee"
-	GasFlag      = "gas"
-	SequenceFlag = "sequence"
+	FlagTo       = "to"
+	FlagAmount   = "amount"
+	FlagFee      = "fee"
+	FlagGas      = "gas"
+	FlagSequence = "sequence"
 )
 
 func init() {
 	flags := SendTxCmd.Flags()
-	flags.String(ToFlag, "", "Destination address for the bits")
-	flags.String(AmountFlag, "", "Coins to send in the format <amt><coin>,<amt><coin>...")
-	flags.String(FeeFlag, "0mycoin", "Coins for the transaction fee of the format <amt><coin>")
-	flags.Int64(GasFlag, 0, "Amount of gas for this transaction")
-	flags.Int(SequenceFlag, -1, "Sequence number for this transaction")
+	flags.String(FlagTo, "", "Destination address for the bits")
+	flags.String(FlagAmount, "", "Coins to send in the format <amt><coin>,<amt><coin>...")
+	flags.String(FlagFee, "0mycoin", "Coins for the transaction fee of the format <amt><coin>")
+	flags.Int64(FlagGas, 0, "Amount of gas for this transaction")
+	flags.Int(FlagSequence, -1, "Sequence number for this transaction")
 }
 
 // runDemo is an example of how to make a tx
 func doSendTx(cmd *cobra.Command, args []string) error {
-	tx := new(btypes.SendTx)
 
 	// load data from json or flags
+	tx := new(btypes.SendTx)
 	found, err := txcmd.LoadJSON(tx)
+	if err != nil {
```

Comment:
Not needed, as if `found == false`, LoadJSON will have done no work and `err` is guaranteed to be `nil`. So I optimized to use only one err checking block.

But this is more clear, and less likely to have subtle errors later introduced.